python3/forwardfinancing.py

```diff
@@ -0,0 +1,116 @@
+import requests
+import os
+import json
+import base64
+
+# Since you don't want to hard-code an api key, read it from environment var
+api_key = os.environ["FORWARD_FINANCING_API_KEY"]
```

Comment:
when not in JSON, be consistent with single vs double quote strings. both are the same but you should pick one and stick with it (I see a few of both and you can use `f-strings` with either I think)